lib/internal/trace_events_async_hooks.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const trace_events = process.binding('trace_events');
+const async_wrap = process.binding('async_wrap');
+const async_hooks = require('async_hooks');
+
+// Use small letters such that chrome://traceing groups by the name.
+// The behaviour is not only useful but the same as the events emitted using
+// the specific C++ macros.
+const BEFORE_EVENT = 'b'.charCodeAt(0);
+const END_EVENT = 'e'.charCodeAt(0);
```

Comment:
extremely minor nit but why not just hard code the char code?

Reply:
This is more readable. Otherwise, they are just magic numbers. And there is basically no extra cost.